docs/source/models/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -180,8 +186,8 @@ We will consider two concepts of SQ-LNS services, including coverage and utiliza
 Possible coverage values include:
 
 - **Uncovered:** Not supplemented by SQ-LNS currently or in the past. Simulants aged 6 months to 5 years are eligible for this category.
-- **Covered:** Actively receiving SQ-LNS supplementation. Simulants aged 6 months to 2 years are eligible for this category. Simulants in this category are subject to the SQ-LNS effects on wasting and stunting.
-- **Received:** No longer actively receiving SQ-LNS supplementation, but did receive SQ-LNS supplementation before the age of 2. Simulants aged 2 to 5 years are eligible for this category. Simulants in this category are subject to the SQ-LNS effects on stunting, but not wasting.
+- **Covered:** Actively receiving SQ-LNS supplementation. Simulants aged 6 months to 18 months are eligible for this category. Simulants in this category are subject to the SQ-LNS effects on wasting and stunting.
+- **Received:** No longer actively receiving SQ-LNS supplementation, but did receive SQ-LNS supplementation before the age of 18 months. Simulants aged 18 months to 5 years are eligible for this category. Simulants in this category are subject to the SQ-LNS effects on stunting, but not wasting.
```

Comment:
how is the evidence base for the assumption that treatment affects stunting for years after it is completed?